server/models/ParticipantsModel/index.js

```diff
@@ -3,6 +3,7 @@ import { ASC } from '../../constants'
 
 const subjects = '$subjects'
 const subject = '$subject'
+const participant = 'participant'
```

Comment:
should this have a `$`? not sure what the siginificance of that character is in the lines above just wanted to call it out

Reply:
The prepended $ signals to mongo that after we retrieved our documents we will be performing operations using the value of the property. 
I've added the $ to the constant to make the distinction in variables.
Good catch 👍 

[8fc6b20](https://github.com/AMP-SCZ/dpdash/pull/561/commits/8fc6b209dd2332cf93ba6881e649bc1bb4a89c3b)